spec/services/estate_visit_query_spec.rb

```diff
@@ -106,6 +106,25 @@
           is_expected.to eq([booked])
         end
       end
+
+      context 'when visits have not been updated within six months' do
+        let!(:old_booked) { FactoryBot.create(:booked_visit, prison: prison, updated_at: 7.months.ago) }
+        let(:prisoner_number) { old_booked.prisoner.number.downcase + ' ' }
```

Comment:
is the `+ ' '` to see if the class trims the input?

Reply:
@StupidCodeFactory - I don't think the trailing white space is required. I've removed both instances and tests pass.